code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -164,7 +164,7 @@
 			. = stored_mmi
 			if(owner.mind)
 				owner.mind.transfer_to(stored_mmi.brainmob)
-			stored_mmi.forceMove(get_turf(src))
+			stored_mmi.forceMove(get_turf(owner))
 			stored_mmi = null
 	..()
```

Comment:
I feel like this issue can also be fixed by moving this `..()` to above the `if` proc. I don't know if this would break anything. Thoughts @maintainers?

Reply:
Nah, leave it like this, I'm 99% sure that this problem was created due to the "organs are no longer in a mob" thing and legacy code that expected them to be

Reply:
Yup, you're correct. In that case, I will keep it as-is